docs/src/dataset_examples.md

```diff
@@ -1,111 +1,167 @@
-# Decision Trees on Real-World Datasets
 
+# Decision Trees and Random Forests on Real-World Datasets
 
-## Classification Tree - Classifying iris types
-In this tutorial, we will demonstrate how to use the package to create a classification tree and apply it to classify the Iris dataset.  
 
-First, ensure to import all the necessary packages: 
+## Setup
 
-```@example 2
-using DataFrames  
-using MLJ: load_iris,unpack,partition
+```
+julia
+using Pkg
+Pkg.activate("DecisionTreeAndRandomForest")
```

Comment:
I don't think that this is necessary. It probably even created the additional folder ^^